src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php

```diff
@@ -63,12 +63,17 @@ protected function cleanParameterBag(ParameterBag $bag)
      * Clean the data in the given array.
      *
      * @param  array  $data
+     * @param  string  $key
      * @return array
      */
-    protected function cleanArray(array $data)
+    protected function cleanArray(array $data, $key = '')
```

Comment:
A parameter name of `$parentKey` might make more sense in the context of the `Closure` below?